jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ByteBufferAccumulator.java

```diff
@@ -18,44 +18,53 @@
 import java.nio.ByteBuffer;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
 
 import org.eclipse.jetty.util.BufferUtil;
+import org.eclipse.jetty.util.Callback;
 
 /**
  * Accumulates data into a list of ByteBuffers which can then be combined into a single buffer or written to an OutputStream.
  * The buffer list automatically grows as data is written to it, the buffers are taken from the
  * supplied {@link ByteBufferPool} or freshly allocated if one is not supplied.
- *
- * The method {@link #ensureBuffer(int, int)} is used to write directly to the last buffer stored in the buffer list,
+ * <p>
+ * The method {@link #accessInternalBuffer(int, int)} can be used access a buffer that can be written to directly as the last buffer list,
```

Comment:
I do not understand the following sentence:
"that can be written to directly _as the last buffer list_"